testing/btest/coverage/bare-load-baseline.test

```diff
@@ -12,5 +13,6 @@
 # @TEST-EXEC: test -e loaded_scripts.log
 # @TEST-EXEC: cat loaded_scripts.log | egrep -v '#' | awk 'NR>0{print $1}' | sed -e ':a' -e '$!N' -e 's/^\(.*\).*\n\1.*/\1/' -e 'ta' >prefix
 # @TEST-EXEC: (test -L $BUILD && basename $(readlink $BUILD) || basename $BUILD) >buildprefix
-# @TEST-EXEC: cat loaded_scripts.log | sed "s#`cat buildprefix`#build#g" | sed "s#`cat prefix`##g" >canonified_loaded_scripts.log
+# @TEST-EXEC: cat loaded_scripts.log | sed "s#`cat buildprefix`#build#g" | sed "s#`cat prefix`##g" >prefix_canonified_loaded_scripts.log
+# @TEST-EXEC: grep -v 'Zeek_AF_Packet' prefix_canonified_loaded_scripts.log > canonified_loaded_scripts.log
```

Comment:
Arne what motivates this filtering, also below in the plugin? It's fine with me, I'm just trying to understand it in relation to the Spicy plugin, where we don't filter either. Is it the conceptual fact that the scripts live outside of Zeek proper?

Reply:
In short: Avoiding baseline differences between Linux and OSX developer systems for those tests in the fairly straight-forward way. On OSX there'll never be AF_Packet scripts loaded while on Linux by default there will.

Spicy is platform-agnostic and build by default everywhere, that's the relation/difference I see.

Though, I've run multiple times into the situation where I used a non-spicy-included Zeek build locally for dev just to be told by CI later the plugin/hooks.zeek baseline is outdated. That test does not run locally if one has spicy disabled and I had wanted to avoid introducing more of that (or not running certain tests on OSX just because AF_Packet shows in the baselines).